ecosystem/indexer-grpc/indexer-grpc-data-service/src/config.rs

```diff
@@ -71,29 +66,33 @@ pub struct IndexerGrpcDataServiceConfig {
     /// Support compressed cache data.
     #[serde(default = "IndexerGrpcDataServiceConfig::default_enable_cache_compression")]
     pub enable_cache_compression: bool,
+    /// Sender addresses to ignore. Transactions from these addresses will not be indexed.
+    #[serde(default = "IndexerGrpcDataServiceConfig::default_sender_addresses_to_ignore")]
+    pub sender_addresses_to_ignore: Vec<String>,
 }
 
 impl IndexerGrpcDataServiceConfig {
     pub fn new(
         data_service_grpc_tls_config: Option<TlsConfig>,
         data_service_grpc_non_tls_config: Option<NonTlsConfig>,
         data_service_response_channel_size: Option<usize>,
-        whitelisted_auth_tokens: Vec<String>,
         disable_auth_check: bool,
         file_store_config: IndexerGrpcFileStoreConfig,
         redis_read_replica_address: RedisUrl,
         enable_cache_compression: bool,
+        blacklist_sender_addresses: Vec<String>,
     ) -> Self {
         Self {
             data_service_grpc_tls_config,
             data_service_grpc_non_tls_config,
             data_service_response_channel_size: data_service_response_channel_size
                 .unwrap_or_else(Self::default_data_service_response_channel_size),
-            whitelisted_auth_tokens,
+            whitelisted_auth_tokens: vec![],
             disable_auth_check,
             file_store_config,
             redis_read_replica_address,
             enable_cache_compression,
+            sender_addresses_to_ignore: blacklist_sender_addresses,
```

Comment:
Want to just call this `blacklist_sender_addresses` or the param to `sender_addresses_to_ignore` just to keep same naming convention?

Reply:
yep; changed to `sender_addresses_to_ignore`. i want to avoid the `blacklist` but forgot to change this one.